client/src/components/AboutArtistTab/AboutArtistTab.tsx

```diff
@@ -0,0 +1,54 @@
+import Box from '@material-ui/core/Box';
+import { PersonalInfo } from '../../interface/PersonalInfo';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import EmailIcon from '@material-ui/icons/Email';
+import PhoneIcon from '@material-ui/icons/Phone';
+
+interface Props {
+  info: PersonalInfo | undefined | null;
+}
+
+const AboutArtistTab = ({ info }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box textAlign="center">
+      {info !== null && info !== undefined ? (
```

Comment:
Eventually we may need to change this logic when we have reviews available to us

Reply:
yes, we never discussed about how we are dealing with reviews. apparently dealt this way.